README.md

```diff
@@ -1,63 +1,17 @@
-# SnarkyJS
+# SnarkyJS &nbsp; [![npm version](https://img.shields.io/npm/v/snarkyjs.svg?style=flat)](https://www.npmjs.com/package/snarkyjs) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](#contributing)
```

Comment:
oof I missed this link to the contributing content. If we decide to use my suggestion to contain the contributing guidelines in a separate file, let's be sure to update this link

Reply:
@mitschabaude  can you update this bit to use the new contributing.md file link?